lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -70,6 +70,26 @@ def request_json_including(required_attributes)
         end
       end
 
+      def publishing_api_has_fields_for_format(format, items, fields)
+        body = items.map { |item|
+          item.with_indifferent_access.slice(*fields)
+        }
+
+        query_params = fields.map { |f|
+          "&fields%5B%5D=#{f}"
```

Comment:
Could there be cases where fields need to be URI escaped? I can't think of any, so this is probably fine.